PennMobile/LaundryPrototype/Models/Laundry.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  Laundry.swift
+//  PennMobile
+//
+//  Created by Nathan Aronson on 10/5/25.
+//  Copyright © 2025 PennLabs. All rights reserved.
+//
+
+struct LaundryHallId: Codable, Hashable {
```

Comment:
I would call this something a bit more descriptive than `LaundryHallId`, but I can't immediately come up with a better name so we can punt this for later